site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -159,8 +159,7 @@ The popup can be registered in the manifest under the `"action"` key.
 }
 ```
 
-Popups can also be set dynamically by calling [`action.setPopup`][18] or
-[`action.setPopup`][19].
+Popups can also be set dynamically by calling [`action.setPopup`][18].
```

Comment:
@dotproto, I'm guessing this isn't a mere duplication. The second link was probably intended to be to something else, but I don't know what it might be.

Reply:
Looks like PR #1139 updated this page to move from `browserAction` and `pageAction` to `action`. I'd bet this was a mostly automated find/replace which lead to duplicate entries appearing next to each other. Removing the extra reference here is fine.